src/app/admin/access-requests/page.simple.test.tsx

```
@@ -0,0 +1,60 @@
+      setRememberMe: vi.fn(),
+    }),
+  }
+})
+
+// Mock Supabase
+vi.mock('@/lib/supabase-client', () => ({
+  getSupabaseClient: () => ({
+    from: () => ({
+      select: () => ({
+        order: () => ({
+          then: (cb: any) => cb({ 
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)